src/main/java/com/icthh/xm/tmf/ms/communication/utils/ApiMapper.java

```diff
@@ -0,0 +1,27 @@
+package com.icthh.xm.tmf.ms.communication.utils;
+
+import com.icthh.xm.tmf.ms.communication.web.api.model.CommunicationMessageCreate;
+import com.icthh.xm.tmf.ms.communication.web.api.model.Receiver;
+import java.util.List;
+import java.util.stream.Collectors;
+import lombok.RequiredArgsConstructor;
+
+public class ApiMapper {
+
+    public static CommunicationMessageCreateWeapper from(CommunicationMessageCreate messageCreate) {
+        return new CommunicationMessageCreateWeapper(messageCreate);
+    }
+
+    @RequiredArgsConstructor
+    public static class CommunicationMessageCreateWeapper {
+        private final CommunicationMessageCreate message;
+        public List<String> getPhoneNumbers() {
+            return message.getReceiver().stream()
+                          .map(Receiver::getPhoneNumber)
+                          .collect(Collectors.toList());
```

Comment:
@sergeysenja1992 is it allowed to send message to the same number? Who should control msisdns uniqueness? It may worth to consider collection to Set.

Reply:
Model no't control uniqueness. It not responsibility of this model.